### PR TITLE
fixed-old-paths

### PR DESCRIPTION
**Pull request by avesda**
I have used a script to find the references of the pages inside all .md files in avm-staging and then matched the diff command output(master..avm-staging) to find the paths that needed to be replaced. Although it fixed most of the cases, there are still some paths that have to be fixed by hand. Most of them are related to the newly added pyteal and removed sdks folders in avm-staging. It has also fixed paths to some of the .img and .png files. 
I'm including a report.txt file that has all the changes listed in it and a deleted_and_added.txt file which is a part of the diff command.
I'm also including a diff output file for files that have been relocated in avm-staging. It has two rows for each case, where 1 indicates the path to the file in master and 2 is the updated path in avm-staging. 

[report.txt](https://github.com/algorand/docs/files/6891457/report.txt)
[deleted_and_added.txt](https://github.com/algorand/docs/files/6891996/deleted_and_added.txt)
[diff.txt](https://github.com/algorand/docs/files/6892033/diff.txt)

### CODE REVIEW

**Comment by yigitguler** [comment]
Is there a way to keep links relative? I think we should use relative links as much as possible. They offer big advantage for future refactors.

**Comment by ejbaran** [comment]
@avesda can you also make sure you sign the CLA? We will need that before we can merge anything in.

**Comment by avesda** [comment]
I have signed the CLA and added a new commit which changes paths to relative.

File: docs/archive/build-apps/apps.md [hunk 1]
```diff
@@ -11,85 +11,85 @@ Payment-based applications have a primary purpose of transferring funds from one
...
 
-In the case of a voting application, a vote is just a specialized transaction where the stateful smart contract increments the vote count for the candidate of choice by the user and also marks the user’s account as having voted (stored as a value in the user’s local state). For more information see the [Stateful Smart Contract](../features/asc1/stateful/index.md) guide.
+In the case of a voting application, a vote is just a specialized transaction where the stateful smart contract increments the vote count for the candidate of choice by the user and also marks the user’s account as having voted (stored as a value in the user’s local state). For more information see the [Stateful Smart Contract](../../get-details/dapps/pyteal/smart-contracts/stateful) guide.
```
Comment (yigitguler) [inline docs/archive/build-apps/apps.md:70]: @avesda  is it possible to make this /stateful/index.md?

Comment (avesda) [reply docs/archive/build-apps/apps.md:70]: It seems like mkdocs automatically detects the index.md and launches it. If I add index.md in the url I will get 404.

**Comment by avesda** [comment]
Conflicts were resolved and a minor fix was applied to remove unnecessary warnings created by mkdocs for having (..) or [..]. Replaced them with (../) and [../]. Ready to merge.

**Review by ejbaran — APPROVED** [review]
LGTM

**Pull request merged**